sdk/agrifood/Azure.Verticals.AgriFood.Farming/src/Azure.Verticals.AgriFood.Farming.csproj

```diff
@@ -5,7 +5,7 @@
     <Version>1.0.0-beta.2</Version>
     <PackageTags>Azure FarmBeats</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <NoWarn>$(NoWarn);AZC0001</NoWarn>
+    <NoWarn>$(NoWarn);AZC0001;AZC0012;AZC0007;AZC0006</NoWarn>
```

Comment:
These warning suppressions must be reverted, errors fixed:
- For AZC0001, please create an issue in in azure-sdk repo to add Azure.Verticals to the list of well-known namespaces
- AZC0006 and AZC0007 should not appear for subclient. For the top-level client, they are valid
- For the AZC0012, single-worded type names require approval. We have to provide better names. See see https://github.com/Azure/autorest.csharp/blob/feature/v3/samples/Azure.Analytics.Purview.Account/Customizations.cs  on how customize type names or change the hierarchy of clients if needed (you don't need to specify ParentClient if existing hierarchy works for you).

@tg-msft , @KrzysztofCwalina - do you think it is ok to merge with these warnings suppressed, assuming that they will be addressed before RTM?

Reply:
Shape of the APIs (subclients, naming, etc.) are OK to release in a preview. We just need to fix before GA. 

Package name issues needs to be resolved before we release the first preview. But it seems like this is not the first preview, so in this particular case, we are good to. 

cc: @tg-msft